<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Медиаплеер 3.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res.qrc">
    <normaloff>:/res/img/16x16/appicon.png</normaloff>:/res/img/16x16/appicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QFrame" name="playbackFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="mediaSBackwardButton">
         <property name="styleSheet">
          <string notr="true">QPushButton:hover {
border: 1px solid #8f8f91;
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res.qrc">
           <normaloff>:/res/img/32x32/skip-backward.png</normaloff>:/res/img/32x32/skip-backward.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mediaBackwardButton">
         <property name="styleSheet">
          <string notr="true">QPushButton:hover {
border: 1px solid #8f8f91;
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res.qrc">
           <normaloff>:/res/img/32x32/backward.png</normaloff>:/res/img/32x32/backward.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mediaPlayButton">
         <property name="styleSheet">
          <string notr="true">QPushButton:hover {
border: 1px solid #8f8f91;
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res.qrc">
           <normaloff>:/res/img/32x32/start.png</normaloff>:/res/img/32x32/start.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mediaPauseButton">
         <property name="styleSheet">
          <string notr="true">QPushButton:hover {
border: 1px solid #8f8f91;
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res.qrc">
           <normaloff>:/res/img/32x32/pause.png</normaloff>:/res/img/32x32/pause.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mediaStopButton">
         <property name="styleSheet">
          <string notr="true">QPushButton:hover {
border: 1px solid #8f8f91;
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res.qrc">
           <normaloff>:/res/img/32x32/stop.png</normaloff>:/res/img/32x32/stop.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mediaForwardButton">
         <property name="styleSheet">
          <string notr="true">QPushButton:hover {
border: 1px solid #8f8f91;
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res.qrc">
           <normaloff>:/res/img/32x32/forward.png</normaloff>:/res/img/32x32/forward.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mediaSForwardButton">
         <property name="styleSheet">
          <string notr="true">QPushButton:hover {
border: 1px solid #8f8f91;
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res.qrc">
           <normaloff>:/res/img/32x32/skip-forward.png</normaloff>:/res/img/32x32/skip-forward.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VolumeButton" name="volumeButton">
         <property name="styleSheet">
          <string notr="true">QToolButton:hover {
border: 1px solid #8f8f91;
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }
QToolButton::menu-indicator { image: none; }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res.qrc">
           <normaloff>:/res/img/32x32/volume-muted.png</normaloff>:/res/img/32x32/volume-muted.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRepeatDelay">
          <number>300</number>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="timeLabel">
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="PositionSlider" name="positionSlider">
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QLabel" name="divLabel">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="8">
     <widget class="QTableView" name="localPlaylistTable">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QLabel" name="durationLabel">
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="6">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>105</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Сервер</string>
    </property>
    <addaction name="loadAllAction"/>
    <addaction name="parseLocalFileAction"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Воспроизведение</string>
    </property>
    <addaction name="sBackwardAction"/>
    <addaction name="backwardAction"/>
    <addaction name="playAction"/>
    <addaction name="pauseAction"/>
    <addaction name="stopAction"/>
    <addaction name="forwardAction"/>
    <addaction name="sForwardAction"/>
   </widget>
   <widget class="QMenu" name="menuService">
    <property name="title">
     <string>Сервис</string>
    </property>
    <addaction name="settingsAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuServer"/>
   <addaction name="menuService"/>
  </widget>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/img/16x16/application-exit.png</normaloff>:/res/img/16x16/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="loadAllAction">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/img/16x16/download.png</normaloff>:/res/img/16x16/download.png</iconset>
   </property>
   <property name="text">
    <string>Загрузить всё с сервера</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="sBackwardAction">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/img/16x16/skip-backward.png</normaloff>:/res/img/16x16/skip-backward.png</iconset>
   </property>
   <property name="text">
    <string>Предыдущий</string>
   </property>
   <property name="toolTip">
    <string>Предыдущий</string>
   </property>
  </action>
  <action name="backwardAction">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/img/16x16/backward.png</normaloff>:/res/img/16x16/backward.png</iconset>
   </property>
   <property name="text">
    <string>Назад 5 сек</string>
   </property>
  </action>
  <action name="playAction">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/img/16x16/play.png</normaloff>:/res/img/16x16/play.png</iconset>
   </property>
   <property name="text">
    <string>Проигрывать</string>
   </property>
  </action>
  <action name="pauseAction">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/img/16x16/pause.png</normaloff>:/res/img/16x16/pause.png</iconset>
   </property>
   <property name="text">
    <string>Пауза</string>
   </property>
  </action>
  <action name="stopAction">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/img/16x16/stop.png</normaloff>:/res/img/16x16/stop.png</iconset>
   </property>
   <property name="text">
    <string>Стоп</string>
   </property>
  </action>
  <action name="forwardAction">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/img/16x16/forward.png</normaloff>:/res/img/16x16/forward.png</iconset>
   </property>
   <property name="text">
    <string>Вперед 5 сек</string>
   </property>
   <property name="toolTip">
    <string>Вперед 5 сек</string>
   </property>
  </action>
  <action name="sForwardAction">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/res/img/16x16/skip-forward.png</normaloff>:/res/img/16x16/skip-forward.png</iconset>
   </property>
   <property name="text">
    <string>Следующий</string>
   </property>
   <property name="toolTip">
    <string>Следующий</string>
   </property>
  </action>
  <action name="parseLocalFileAction">
   <property name="text">
    <string>Загрузить локальный файл</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PositionSlider</class>
   <extends>QSlider</extends>
   <header>src/ui/positionslider.h</header>
  </customwidget>
  <customwidget>
   <class>VolumeButton</class>
   <extends>QToolButton</extends>
   <header>src/ui/volumebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
